<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_apps_menu</class>
 <widget class="QWidget" name="create_apps_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>550</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="path_input">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>290</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>140</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="path_label">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>240</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Path</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="right_button">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>430</y>
     <width>113</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>30</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="left_button">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>430</y>
     <width>115</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>30</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>30</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
